lib/saddle/endpoint.rb

```diff
@@ -26,7 +26,7 @@ def request(method, action, params={}, options={})
       # Augment in interesting options
       options[:saddle] ||= {}
       options[:saddle] = {
```

Comment:
Same here. Maybe squash the options down onto 'options', rather than `options[:saddle]`?